src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/SomaticGenotypingEngineUnitTest.java

```diff
@@ -69,7 +85,7 @@ Object[][] missingAFData() {
 
     @Test(dataProvider = "missingAFData")
     public void testGetGermlineAltAlleleFrequenciesWithMissingAF(final VariantContext vc) {
-        final double[] result = SomaticGenotypingEngine.getGermlineAltAlleleFrequencies(vc.getAlleles(), Optional.of(vc), DEFAULT_AF);
```

Comment:
Add a test (here or separate) that handles the edge case involving the wrong-length AF list for multiallelics.

Reply:
done